src/Assignment4/CardComponent/index.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
 
+import userIcon from "../../logo/userIcon.png";
 import "./styles-card.css";
 
 const CardComponent = ({ temmatesDetails }) => (
   <div className="CardWrapper">
-    <img src={temmatesDetails.imageSource} className="imageClass" />{" "}
+    <img
```

Comment:
Can destructure it more like this: 
{temmatesDetails: { name, location, company}}

Reply:
good one. thanks